packages/bierzo-wallet/src/store/balances/index.unit.spec.ts

```diff
@@ -45,11 +45,9 @@ withChainsDescribe('Tokens reducer', () => {
     store.dispatch(addBalancesAction(tokens));
 
     const balances = store.getState().balances;
-    const ethBalance = {
-      fractionalDigits: 18,
-      quantity: '100000000000000000000',
-      tokenTicker: 'ETH',
-    };
-    expect(balances).toEqual({ ETH: ethBalance });
+    expect(balances.ETH).toBeDefined();
+    expect(balances.ETH.fractionalDigits).toEqual(18);
+    expect(balances.ETH.quantity).toMatch(/^[0-9]{10,22}$/);
```

Comment:
Quantity can by any number from 10 to 22 symbols length? I think it is much better to test it against string at least for readability.

Reply:
The problem was that some other tests send tokens from this account, such that the quantity changes from time to time. Instead of one fixed value we just require _some large number_ now